### PR TITLE
Visitors addition

### LINKED ISSUE #977: ✨[Feature Request]: Visitors addition
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

Addition of no of visitors to the index.hmtl page.

### Add ScreenShots

image

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Anjaliavv51**
#977

## Description

[Please include a brief description of the changes or features added]

## Related Issues

[Cite any related issue(s) this pull request addresses. If none, simply state “None”]
- Closes #977

## Type of PR

- [ ] Bug fix
- [X] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Checklist

- [X] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [X] I have updated my branch and synced it with project `main` branch before making this PR
- [X] I have performed a self-review of my code
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

File: index.html [hunk 1]
```diff
@@ -1021,8 +1021,17 @@ <h2>Thank you for your Feedback! 💚</h2>
...
   </main><!-- End #main -->
+  <br>
+  <link rel="stylesheet" href="./assets/css/visitors.css">
   <!-- ############### Footer ############### -->
-  
+  <div class="visitor-counter">
+    <div>Visitor</div>
+    <div class="website-counter"></div>
+  </div>
+  <script src="path/to/visitorCounter.js"></script>
```
Comment (varshith257) [inline index.html:1031]: I think this path need to be updated

**Review by varshith257 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Anjaliavv51) [reply index.html:1031]: Why ? @varshith

Comment (varshith257) [reply index.html:1031]: Where is path/to/ .... Directory in our project?

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR.

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Pull request merged**